### PR TITLE
Refactor / Fix bugs

### PR DESCRIPTION
**Pull request by axelsheva**

### CODE REVIEW

File: core/loader.ts [hunk 1]
```diff
@@ -15,14 +15,19 @@ import RoundService from 'core/service/round';
 import BlockService from 'core/service/block';
 import RoundRepository from 'core/repository/round';
 
+// @ts-ignore
+BigInt.prototype.toJSON = function () {
```
Comment (alex-arugon) [inline core/loader.ts]: Why here? Why do we need this? Just for logging?

File: core/service/round.ts [hunk 2]
```diff
@@ -208,19 +207,33 @@ class RoundService implements IRoundService {
         const mySlot = this.getMyTurn();
         if (mySlot) {
             // start forging block at mySlotTime
-            const cellTime = SlotService.getSlotTime(mySlot - SlotService.getSlotNumber());
-            logger.info(`${this.logPrefix}[generateRound] Start forging block to: ${mySlot} after ${cellTime} seconds`);
-            createTaskON('BLOCK_GENERATE', cellTime, {
-                timestamp: SlotService.getSlotTime(mySlot),
-                keyPair: this.keyPair,
-            });
+            let cellTime = SlotService.getSlotRealTime(mySlot) - new Date().getTime();
```
Comment (alex-arugon) [inline core/service/round.ts]: callTime? timeToCall? queue? turn?

Comment (Verbalist) [reply core/service/round.ts]: cellTime

File: core/loader.ts (hunk 1 above)
Comment (axelsheva) [reply core/loader.ts]: It is serializer

**Pull request merged**